crd/multitenancy/api/v1alpha1/multitenantpodnetworkconfig.go

```diff
@@ -45,8 +45,22 @@ type MultitenantPodNetworkConfigSpec struct {
 	PodNetwork string `json:"podNetwork"`
 	// name of the requesting cx pod
 	PodName string `json:"podName,omitempty"`
+	// MAC addresses of the IB devices to use for a pod
+	// +kubebuilder:validation:Optional
+	IBMACAddresses []string `json:"IBMACAddresses,omitempty"`
```

Comment:
TODO: for CNS API PR - we can validate the mac-format passed in here as part of request body validation